source/docs/steeltoe-connectors.html.markdown

```diff
@@ -405,46 +413,49 @@ If you are using a different service, adjust the `create-service` command to fit
 > cf services
 ```
 
-### 2.1.3 Publish and Push Sample, observe logs
+### 2.1.3 Publish and Push Sample
+
+See [Publish Sample](#publish-sample) for instructions on how to publish and push this sample to either Linux or Windows.
 
-See [Publish Sample](#publish-sample) and the sections that follow for instructions on how to publish and push this sample to either Linux or Windows. Optionally use the `cf logs` command to see log output.
+### 2.1.4 Observe Logs
+
+Optionally, you can use the `cf logs` command to see log output.
 
 ### 2.1.4 What to expect
 
-At this point, the app is up and running. Upon startup it inserts a couple rows into the bound PostgreSQL database.
+At this point, the app is up and running. On startup, it inserts a couple of rows into the bound PostgreSQL database.
 
 To display those rows, click on the `Postgres Data` link in the top menu.
 
 ### 2.1.5 Understand Sample
 
-Each of the samples were created from the .NET Core tooling `mvc` template ( i.e. `dotnet new mvc` ) and then modified to include the Steeltoe framework.
+Each of the samples were created from the .NET Core tooling `mvc` template (`dotnet new mvc`) and then modified to include the Steeltoe framework.
 
 To gain an understanding of the Steeltoe related changes to the generated template code, examine the following files:
 
-* `PostgreSql.csproj` - Contains a `PackageReference` for Steeltoe NuGet `Steeltoe.CloudFoundry.ConnectorCore`
-* `PostgreEFCore.csproj` - Contains a `PackageReference` for Steeltoe NuGet `Steeltoe.CloudFoundry.Connector.EFCore`
-* `Program.cs` - added `.UseCloudFoundryHosting()` for dynamic port binding and `.ConfigureAppConfiguration( -> .AddCloudFoundry()` to read `VCAP_SERVICES` when pushed to Cloud Foundry
-* `Startup.cs` - Code added to the `ConfigureServices()` method to add a `NpgsqlConnection` or a `DbContext`, depending on the application, to the service container.
-* `HomeController.cs` - Code added to inject a `NpgsqlConnection` or `DbContext` into the Controller and obtain data from the database for the view.
-* `PostgresData.cshtml` - The view used to display the PostgreSQL data values.
-* `Models folder` - contains code to initialize the database and also the `DbContext` for PostgreEFCore sample.
+* `PostgreSql.csproj`: Contains a `PackageReference` for the Steeltoe NuGet `Steeltoe.CloudFoundry.ConnectorCore`.
+* `PostgreEFCore.csproj`: Contains a `PackageReference` for Steeltoe NuGet `Steeltoe.CloudFoundry.Connector.EFCore`.
```

Comment:
Maybe we should just remove `Steeltoe NuGet` completely and rely on Steeltoe being in the name of what's referenced